apis/v1alpha1/nginxproxy_types.go

```diff
@@ -32,6 +32,11 @@ type NginxProxySpec struct {
 	//
 	// +optional
 	Telemetry *Telemetry `json:"telemetry,omitempty"`
+	// DisableHTTP2 defines if http2 should be disabled for all servers.
+	// Default is false, meaning http2 will be enabled for all servers.
+	//
+	// +optional
+	DisableHTTP2 bool `json:"disableHTTP2,omitempty"`
```

Comment:
What do you think about `http2` to field set to true with the default in CRD?

in my view, it is easier to work with fields without negatives

Reply:
The issue with that is this CRD doesn't exist by default. So it seems like if it has a default value of "true" in it, then that value would be false if the CRD doesn't exist.

Reply:
@pleshakov normally I would agree with you, but in this case it seems unintuitive as, as @sjberman points out, the resource doesn't exist by default. So we could have a default of true in the CRD, but then we would also need to assume true when the CRD doesn't exist - which is the same result as what we are doing now, but feels less explicit.